src/js/rtc_session.js

```diff
@@ -685,7 +699,7 @@ export default class RtcSession {
      * Optional. RtcSession will grab input device if this is not specified.
```

Comment:
Also make it clear that we only support single track audio and/or single track video